src/rsz/test/repair_fanout7.ok

```diff
@@ -11,7 +11,8 @@ load DFF_X1 CK D 300000 metal3 1000
 [INFO ODB-0132]     Created 2 special nets and 306 connections.
 [INFO ODB-0133]     Created 4 nets and 307 connections.
 worst slack -23.63
-[INFO RSZ-0043] Swapped pins on 1 instances.
-[INFO RSZ-0049] Cloned 1 instances.
+[INFO RSZ-0040] Inserted 34 buffers.
+[INFO RSZ-0041] Resized 139 instances.
+[INFO RSZ-0049] Cloned 6 instances.
 [WARNING RSZ-0062] Unable to repair all setup violations.
-worst slack -0.06
+worst slack -3.23
```

Comment:
Is this degradation expected?

Reply:
The earlier result was not correct. The comparison is with repair_fanout8 (which shows something like -5 in wns). So yes. This degradation can be waived.